### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by lbonaldo**
This PR updates the documentation for EP following comments from users.
- add instructions on how to install git
- include interface and data setup as part of `Installation` section
- add `Uninstall` instructions
- minor changes to `xlwings.conf` instructions

### CODE REVIEW

File: docs/source/setup.rst [hunk 1]
```diff
@@ -29,7 +29,14 @@ Model outputs consist of large csv files that have been designed to work well wi
 
 Installation
 ============
-EnergyPATHWAYS can be installed using the ``environment.yml`` file provided in the repository. This file contains all the necessary packages to run EnergyPATHWAYS. To install the environment, navigate to the directory containing the ``environment.yml`` file and run the following commands::
+First download the EnergyPATHWAYS code from the `EnergyPATHWAYS GitHub repository <https://github.com/EvolvedEnergyResearch/EnergyPATHWAYS>`_ either by downloading the zip file or by cloning the repository using git::
+
+    $ git clone https://github.com/EvolvedEnergyResearch/EnergyPATHWAYS
+
+.. Note::
```
Comment (gschivley) [inline docs/source/setup.rst:36]: Does the EP code run if git is not installed? I thought git was a requirement.

File: docs/source/setup.rst [hunk 2]
```diff
@@ -49,48 +56,61 @@ After running the setup script, the excel interface will be able to call the Ene
...
+1. Copy the `EP interface` folder outside of EnergyPathways main folder to the working directory. 
+2. Open the `scenario_builder.xlsm` file and enable macros.
+3. Setup the `xlwings.conf` sheet in the `scenario_builder.xlsm` file to point to the conda environment you created in the previous step. This step is different for Windows and MacOS users:
 
-For Windows users
-^^^^^^^^^^^^^^^
+    **For Windows users**
 
-1. (Optional, **depreciated**) Install the ``xlwings`` Excel Add-in by running the following command in the terminal::
+    1. (Optional, **depreciated**) Install the ``xlwings`` Excel Add-in by running the following command in the terminal::
```
Comment (gschivley) [inline docs/source/setup.rst]: I wrote this part, but think we should remove it. Listing a step that we don't want users to do is going to be confusing.

File: docs/source/setup.rst (hunk 1 above)
Comment (lbonaldo) [reply docs/source/setup.rst:36]: Yes, I added it to the list of deps in the `environment.yml`. If git is not found, then it will be installed by conda.

Comment (lbonaldo) [reply docs/source/setup.rst:36]: Thanks for spotting this!

File: docs/source/setup.rst (hunk 2 above)
Comment (lbonaldo) [reply docs/source/setup.rst]: I removed it.

**Pull request merged**